purchase.adoc

```diff
@@ -1,675 +1,885 @@
-## Purchase API
+= Purchase API Reference Guide
+:toc: right
+:imagesdir: assets/images
 
-### Service Description
-The purchase API provides information about purchases made through iZettle.
+== About Purchase API
+Provides read only information about purchases made through Zettle.
+Purchase represents a receipt, it's immutable and contains information about items that were sold, one or several payments received, taxes, discounts etc.
+Refunds are represented by the same model but with negative amounts and refund related flags/fields.
```

Comment:
negative _quantity_ not amount (amount == value and quantity == number of units)

Reply:
So it was changed right?
We have examples from january this year with negative amounts

Reply:
Should i change examples with refunds?

Reply:
Ping @izjorgen

Reply:
The amount for refund purchases are negative, that is correct. Maybe @izjorgen was thinking of the products level where negative value is represented with negative quantity on refund purchases.